libd-llvm/src/d/llvm/type.d

```diff
@@ -103,7 +104,17 @@ final class TypeGen {
 				return LLVMPointerType(LLVMInt8TypeInContext(llvmCtx), 0);
 		}
 	}
-	
+
+	LLVMTypeRef getSizeT() {
+		static LLVMTypeRef sizeT;
```

Comment:
That is a terrible idea. Just call LLVMIntPtrTypeInContext when needed, just avoid to call it more than once.